pydra/engine/specs.py

```diff
@@ -22,6 +27,9 @@ class SpecInfo:
 class BaseSpec:
     """The base dataclass specs for all inputs and outputs"""
 
+    def collect_additional_outputs(self, input_spec, inputs, output_dir):
+        return {}
+
     @property
     def hash(self):
```

Comment:
shouldn't the hash take `output_field_name` into account when computing a checksum?

Reply:
maybe it should...